### ISSUE TITLE
Multiple extra taps required to bring up username options

### ISSUE THREAD

**Issue by theridon** [twireapp/Twire#383]
When tapping on somebody's username in chat to bring up the options (mention, duplicate message), a single tap does not work. You have to press a second time just for it to show up. Is it possible to fix this?

**Comment by glokta1** [comment]
Can you confirm this happens only when tapping a user for the first time and works normally for subsequent taps?

**Comment by theridon** [comment]
> Can you confirm this happens only when tapping a user for the first time and works normally for subsequent taps?

No, I have to double tap every time, even if I've tapped before, always requires two taps to work.
